diff --git a/node_modules/react-to-print/lib/index.js b/node_modules/react-to-print/lib/index.js
index 4719a6b..00b7218 100644
--- a/node_modules/react-to-print/lib/index.js
+++ b/node_modules/react-to-print/lib/index.js
@@ -2,7 +2,6 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.useReactToPrint = void 0;
 const React = require("react");
-const react_dom_1 = require("react-dom");
 const Print_1 = require("./Print");
 const build-print-window_1 = require("./utils/build-print-window");
 const copy-styles_1 = require("./utils/copy-styles");
@@ -107,11 +106,7 @@ const useReactToPrint = (props) => {
         // Some environments such as JSDom don't have a document object
         // https://jestjs.io/docs/en/configuration#testenvironment-string
         if (typeof document === "undefined") {
-            if (process.env.NODE_ENV !== "production") {
-                if (!jest) {
-                    log("error", '"react-to-print" is not designed for environments that do not have a `document` object, such as Node. Please use this library inside a browser environment.');
-                }
-            }
+            // DO nothing
             return;
         }
         // We have a page to print, so stop checking for one
@@ -155,7 +150,7 @@ const useReactToPrint = (props) => {
     }, [props.pageStyle]);
     const handlePrint = React.useCallback(() => {
         const { onBeforeGetContent, onAfterPrint, onBeforePrint, bodyClass = "" } = props;
-        const contentEl = react_dom_1.findDOMNode(contentRef.current);
+        const contentEl = contentRef.current;
         if (onBeforeGetContent) {
             const onBeforeGetContentPromise = onBeforeGetContent();
             if (onBeforeGetContentPromise && typeof onBeforeGetContentPromise.then === "function") {
